packages/mobile/src/screens/favorites-screen/AlbumsTab.tsx

```diff
@@ -55,18 +67,33 @@ export const AlbumsTab = () => {
             (trackIds.length === 0 ||
               trackIds.some((t) => {
                 return (
+                  offlineDownloadStatus &&
                   offlineDownloadStatus[t.toString()] ===
-                  OfflineDownloadStatus.SUCCESS
+                    OfflineDownloadStatus.SUCCESS
                 )
               }))
           )
         }
         return true
       })
     },
-    [filterValue, isReachable, isOfflineModeEnabled, isDoneLoadingFromDisk]
+    [
+      filterValue,
+      isReachable,
+      isOfflineModeEnabled,
+      offlineDownloadStatus,
+      isDoneLoadingFromDisk
+    ]
   )
 
+  const numPlayableTracksMap = useMemo(() => {
```

Comment:
could this just be trackCount, and just pass that into the collectionList? it seems the collection list does the manual work of extracting the num, but could be cleaner to just pass a number into the list?

Reply:
@dylanjeffers  well the collection list takes a list of collections, so we can't just pass a single number (since each collection has a different number of tracks). but i feel like im misunderstanding what you're saying?

Reply:
No worries, maybe a quick huddle?

Reply:
💯, pretty sure i had the same thought when writing this lol